src/losses/typeface_perceptual.py

```diff
@@ -0,0 +1,51 @@
+import torch
+import torchvision
+from src.disk import disk 
+from pathlib import Path
+class TypefacePerceptualLoss(torch.nn.Module):
+    def __init__(self, model_remote_path = 'checkpoints/typeface/2/model', model_local_path = 'typeface', resize=True):
+        super(TypefacePerceptualLoss, self).__init__()
```

Comment:
`super().__init__()`

Reply:
I think we can leave it like this for now and you will change it with other perceptual losses later